### DISCUSSION TITLE
what rule or thing changes in vscode-notebooks "#|" to "# |" automatically

### DISCUSSION THREAD

**Discussion by bugsbunny88** [quarto-dev/quarto-cli#9137] [Q&A] [answered]
### Description

I'm just a bit confused if anyone has ran into this, I think its Black/Ruff that might be doing this but I don't know how to fix or add an exception to this rule and what category it falls under but it ends up putting a space between "# and |" ;

**Comment by cscheid** [comment, marked as answer]
Yes, unfortunately that's something that Black currently does. We've contacted them about the possibility of adding flexibility to comment formatting rules and they weren't very receptive to it so we don't have a great workaround to give you at the moment.

**Reply by bugsbunny88** [reply]
ok thanks, its been driving me crazy and I figured I might as well as the plain question . It's okay , that just means we need more wider adoption as a community . The larger the voice > the more receptive the counter party might be . Ruff guys over at Astral (might have a nifty workaround , something to just ping them with plz and ty)

**Comment by cderv** [comment]
For reference, related issue on Quarto linking to some issues on black side. 
* https://github.com/quarto-dev/quarto/issues/135

There was even a PR at some point
* https://github.com/psf/black/pull/3561 

but its related issue was closed 
* https://github.com/psf/black/issues/3557
